SAP S/4HANA OData Cloud Guide 1.2.0 M4 (#495)
mulesoft/docs-connectors: modules/ROOT/pages/sap/sap-s4hana-cloud-connector.adoc

= SAP S/4HANA OData Cloud Connector - Mule 4
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../../assets/images/

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]

> SAP S/4HANA Cloud Connector v1.2

Reviewer: Add Odata?
```suggestion
SAP S/4HANA Odata Cloud Connector v1.2
```

> Anypoint Connector for SAP S/4HANA OData Cloud (S/4HANA OData Cloud Connector) provides full support for 

Reviewer: Should we add "SAP" to the abbreviated name as well?
```suggestion
Anypoint Connector for SAP S/4HANA OData Cloud (SAP S/4HANA OData Cloud Connector) provides full support for 
```

the query, get, create, update, delete entities, and execute functions using the OData API exposed by SAP S/4HANA.

> The connector lets you perform operations to:

Reviewer: lets, allows > enables
```suggestion
The connector enables you to perform operations to:
```

* Create, update, and delete entities

> * Retrieve a single entity or query multiple entities

Reviewer: Do you suppose that this should be two list items, since both retrieve and query, and single and multiple, are different?

* Invoke a remote function

Release Notes: xref:release-notes::connector/sap-s4-hana-connector-release-notes-mule-4.adoc[SAP S/4HANA OData Cloud Connector Release Notes]

== Prerequisites

> * Familiarity with S/4HANA OData, Mule, Anypoint Connectors, and the Anypoint Studio essentials.

Reviewer: Add SAP? and other stuff:
```suggestion
* Familiarity with SAP S/4HANA OData, Mule runtime engine (Mule), Anypoint Connectors, and Anypoint Studio essentials
```

|===
|Studio Field |Description
|Protocol |Protocol selected for the HTTP endpoint. This can be HTTP or HTTPS.
|Host |IP address at which the Mule app listens for requests.
|Port |Port address at which the Mule app listens for requests.
|Base Path |Path at which the Mule app listens for requests.
|===
+

> . Select the plus sign to add a component.

Reviewer: ```suggestion
. Select the plus sign (+) to add a component.
```

[%header,cols="30s,70a"]
|===
|Studio Field |Description

> |Discovery Service Enabled |Feature switch for service discovery. If disabled, Service discovery relative path is not applied.

Reviewer: ```suggestion
|Discovery Service Enabled |Feature switch for service discovery. If disabled, the service discovery relative path is not applied.
```

|Proxy configuration |Configuration for executing requests through a proxy.
|===
+

> . Click OK to save the global connector configuration.

Reviewer: Why did you change the tagging for UI control label?
